manual_user_install.sh

```diff
@@ -0,0 +1,237 @@
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-personal
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-naptime
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-date-time
+    echo
+    echo "Done installing OVOS core"
+    }
+
+function install_systemd (){
+    echo
+    echo "Installing systemd files"
+    echo
+
+    cd $HOME/raspbian-ovos/stage-ovos/00-system/files
+    sed -i s/multi-user/default/g ovos.service
+    for f in *.service ; do
+      sed -i s,/usr/libexec,/home/ovos/.local/bin,g $f
+      # extend the timeouts
+      # sed -i s/=1m/=2m/g $f
+    done
+
+    # install the hook files
+    cp ovos-systemd*  $HOME/.local/bin/
+    chmod +x $HOME/.local/bin/ovos-systemd*
+
+    # sdnotify is required
+    pip3 install sdnotify
+
+    # install the service files
+    if [[ ! -d $HOME/.config/systemd/user ]]; then
+        mkdir -p $HOME/.config/systemd/user
+    fi
+    sed -i s/User=root/#User=root/g ovos-admin-phal.service
+    cp *.service $HOME/.config/systemd/user/
+
+    if [[ $enabled == "YES" ]]; then
+        echo
+        echo "Enabling service files"
+        echo
+        systemctl --user enable ovos
+        systemctl --user enable ovos-messagebus
+        systemctl --user enable ovos-dinkum-listener
+        systemctl --user enable ovos-audio
+        systemctl --user enable ovos-skills
+        systemctl --user enable ovos-phal
+        systemctl --user enable ovos-admin-phal
+        systemctl --user daemon-reload
+    fi
+    echo
+    echo "Done installing systemd files"
+    echo
+    }
+
+function install_extra_skills (){
+    echo
+    echo "Installing extra skills"
+    echo
+    # from NeonGeckoCom
+    pip3 install git+https://github.com/NeonGeckoCom/skill-user_settings
+    pip3 install git+https://github.com/NeonGeckoCom/skill-spelling
+    pip3 install git+https://github.com/NeonGeckoCom/skill-local_music
+    pip3 install git+https://github.com/NeonGeckoCom/skill-caffeinewiz
+
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-weather
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-hello-world
+
+    # common query
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-ddg
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-wolfie
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-wikipedia
+
+    # fallback
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-fallback-chatgpt
+
+    # OCP
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-news
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-somafm
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-youtube-music
+
+    # non pip3 skills
+    if [[ ! -d $HOME/.local/share/mycroft/skills ]]; then
+        mkdir -p $HOME/.local/share/mycroft/skills
+    fi
+    cd $HOME/.local/share/mycroft/skills
+    # jarbasskills
+    git clone https://github.com/JarbasSkills/skill-icanhazdadjokes.git
```

Comment:
This can be installed with pip now

Reply:
I'm inclined to leave it the way you had it and use it as a teaching tool.  If the icanhazdadjokes git is going to stick around, we can use it as a concrete example of how to load non-pip skills.  See the next version.

Reply:
The idea is to get away from non-pip installed skills.  I don't think we should provide and example

Reply:
Got it. KISS. Done.